app/models/post.rb

```diff
@@ -9,8 +9,8 @@ class Post < ApplicationRecord
 
   after_save :update_posts_counter
 
-  def recent_comments
-    comments.order(created_at: :desc).limit(5)
+  def most_recent_five_comments
+    comments.includes(:user).order(created_at: :desc).limit(5)
```

Comment:
- [ ] Kindly make sure to solve the N+1 problem from the posts controller and the user controller not in the models, as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=Make%20sure%20that%20the%20N%2B1%20problem%20is%20solved%20when%20fetching%20all%20posts%20and%20their%20comments%20for%20a%20user.) requirement.